activerecord/lib/active_record/migration.rb

```diff
@@ -1031,20 +1025,20 @@ def schema_migrations_table_name
       end
       deprecate :schema_migrations_table_name
 
-      def get_all_versions(connection = Base.connection)
```

Comment:
The `connection` variable was passed to method, but it wasn't being used at all. Hence the refactor.